fish/Containerfile

```diff
@@ -0,0 +1,13 @@
+# Get RHCOS base image of target cluster `oc adm release info --image-for rhel-coreos`
+# This example uses rhel-coreos image from 4.13.0-rc.0
+FROM quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:42e45dd0ba1be3d2681972d6d38c42008c70ddb8c5a96f1f770a11f9bbfb048f
+
+# RHEL entitled host is needed here to access RHEL packages
+# Create symlink to have RHEL entitlement and redhat.repo available at correct location inside container build
+RUN ln -s /run/secrets/redhat.repo /etc/yum.repos.d/redhat.repo && \
+    ln -s /etc/pki/entitlement-host/ /etc/pki/entitlement && \
```

Comment:
This is a good catch. The container image is missing the requisite symlinks that we include in our standard RHEL/UBI container images. 
```
# podman run --rm -ti ubi9 ls -l /etc/pki/entitlement-host /etc/rhsm-host
lrwxrwxrwx. 1 root root 32 Jan 17 08:26 /etc/pki/entitlement-host -> /run/secrets/etc-pki-entitlement
lrwxrwxrwx. 1 root root 17 Jan 17 08:26 /etc/rhsm-host -> /run/secrets/rhsm
```

I think we'll want the CoreOS team to add at least the following symlinks to the image when it's built:
```
ln -s /run/secrets/etc-pki-entitlement /etc/pki/entitlement-host
ln -s /run/secrets/rhsm /etc/rhsm-host
```

Reply:
Thanks, filed https://issues.redhat.com/browse/OCPBUGS-11181